### PR TITLE
Add MapComplete

### PR DESCRIPTION
**Pull request by pietervdvn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ This list contains projects using OpenStreetMap data for creative purposes, as w
 
 * [iD](http://www.openstreetmap.org/edit?editor=id) - JavaScript based editor for the web browser with a wide array of presets. ([Source Code](https://github.com/openstreetmap/iD))
 * [Level0](http://level0.osmz.ru/) - Text based editor that's useful in specific corner cases. ([Source Code](https://github.com/zverik/level0))
+* [MapComplete](https://mapcomplete.osm.be) - an easy to use map viewer and editor, based on thematic maps. Mobile friendly, has simple questions and picture support.
```
Comment (Unkn0wnCat) [inline README.md:51]: Please add new entries at the bottom of their respective categories.

**Review by Unkn0wnCat — CHANGES_REQUESTED** [review]
Thanks for your PR, MapComplete is a great fit for the list. However please see my attached comments.

File: README.md (hunk 1 above)
Comment (pietervdvn) [reply README.md]: Ok, done

**Comment by Unkn0wnCat** [comment]
Just noticed Map Complete is already listed under "Tasking Managers", so I'll close this PR

If you feel the categorization of Map Complete is to be improved, please open a [discussion topic](https://github.com/osmlab/awesome-openstreetmap/discussions/new)

PS: Don't take this as a discouragement! It's amazing you opened a PR 👍🏼

**Pull request closed without merging**